DigitalOffice/Tests/UserServiceTests/ValidatorsTests/RegisterRequestValidatorTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using FluentValidation;
+using NUnit.Framework;
+using UserService.RestRequests;
+using UserService.Validators;
+
+namespace UserServiceUnitTests.ValidatorsTests
+{
+    public class RegisterRequestValidatorTests
+    {
+        private IValidator<UserCreateRequest> validator = new RegisterRequestValidator();
+
+        [Test]
+        public void EmptyFirstName()
+        {
+            var request = new UserCreateRequest
+            {
+                FirstName = "",
+                LastName = "Example",
+                MiddleName = "Example",
+                Email = "Example@gmail.com",
+                Status = "Example",
+                Password = "Example"
+            };
+            Assert.Throws<ValidationException>(() => validator.ValidateAndThrow(request), "First name is empty.");
+        }
+
+        [Test]
+        public void FirstNameContainsNumbers()
```

Comment:
FirstNameContainsNumbers... So how should the programm react? I'd call this and all the test methods like "ShouldDoSomethingIfSomethingHappened"

Reply:
fixed